ts/src/bitget.ts

```diff
@@ -2623,7 +2623,7 @@ export default class bitget extends Exchange {
         //
         const marketId = this.safeString (ticker, 'symbol');
         const close = this.safeString (ticker, 'lastPr');
-        const timestamp = this.safeInteger (ticker, 'ts');
+        const timestamp = this.omitZero (this.safeInteger (ticker, 'ts'));
```

Comment:
omitZero assumes the input is a string 
![image](https://github.com/ccxt/ccxt/assets/43336371/628583ca-9763-4bc9-ae26-511d88f7eb66)

Reply:
update was pushed, now passes